tsptw_data_dt.h

```diff
@@ -264,6 +272,23 @@ class TSPTWDataDT {
     return tsptw_clients_[i.value()].quantities;
   }
 
+  std::vector<int64>* MaxTimes(const ortools_vrp::Matrix& matrix) const {
+    int64 max_row;
+    int32 size_matrix             = sqrt(matrix.time_size());
+    std::vector<int64>* max_times = new std::vector<int64>();
```

Comment:
Is there a specific reason to use a pointer `*` instead of a simple reference `&`? This leads to a memory leak if not `deleted` manually.

Reply:
my bad, with c++11 move semantics there is no need to bother with pointer or addresses -->  https://stackoverflow.com/a/15704602/1200528

Reply:
As seen together, it is recommended to return vector by value